src/store/actions.js

```diff
@@ -556,9 +558,6 @@ export default {
 		} finally {
 			showNewMessagesNotification(newMessages)
 		}
-	},
-	toggleEnvelopeFlagged({ commit, getters }, envelope) {
```

Comment:
I think those three lines should be restored

Reply:
😬